app/dashboard/_components/cards/InclusionScore/InclusionScoreTimeline.tsx

```diff
@@ -0,0 +1,79 @@
+'use client'
```

Comment:
empty line between use client and imports

Reply:
should have seen this in changes.